spec/features/users_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+feature "User creates an account" do
```

Comment:
New line between 1-2?

Reply:
Fixed up commit (263fba0) in `--fixup` in sha: d234d15